gensim/corpora/wikicorpus.py

```diff
@@ -214,9 +214,9 @@ def extract_pages(f, filter_namespaces=False):
             title = elem.find(title_path).text
             text = elem.find(text_path).text
 
-            ns = elem.find(ns_path).text
-            if filter_namespaces and ns not in filter_namespaces:
-                text = None
+            # ns = elem.find(ns_path).text
```

Comment:
@isohyt  "I've written online word2vec tutorial using 2 different wikipedia dumps"
Can you please provide the link to the tutorial? Thanks!